doc/source/dev/core-dev/licensing.rst.inc

```diff
@@ -14,6 +14,8 @@ custom licenses that require attribution/citation or prohibit use for
 commercial purposes.
 
 PRs are often submitted with content copied or derived from unlicensed code.
+For instance, code published on StackOverflow is covered by a CC-BY-SA license,
```

Comment:
The previous sentence probably should be reworded, as this is an instance of _licensed_ code, not unlicensed code. Maybe "unlicensed code or code from a default license that is not compatible with SciPy's license. For instance, ..."

Reply:
I added your suggestion, thanks @rkern.